package.json

```diff
@@ -77,7 +77,7 @@
     "test:prerender:browser": "REACT_APP_ENV=test SERVER_MODE=built jest --testPathPattern=\"(\\.|/)browserspec\\.tsx?\" -i",
     "test:prerender:screenshots": "REACT_APP_ENV=test SERVER_MODE=built jest --testPathPattern=\"(\\.|/)screenshotspec\\.tsx?\"",
     "test:lighthouse-manual": "REACT_APP_ENV=test lighthouse --view --config-path=./src/test/audits/index.js",
-    "analyze:bundle": "react-scripts build --stats && webpack-bundle-analyzer build/bundle-stats.json -m static -r build/bundle-stats.html -O",
+    "analyze:bundle": "react-scripts build && source-map-explorer 'build/static/js/*.js' -m && source-map-explorer 'build/static/js/*.js' -m --html build/bundle-stats.html",
```

Comment:
is calling `source-map-explorer` 2 times necessary?
also is this script now checking only /static/js/... ? is this the same to what was the as previous script doing?

Reply:
I'm going to change this script. First of all, I wanted to know if we would like to work with `source-map-explorer`. Previous version was creating `json` and `html` result file.

> also is this script now checking only /static/js/... ? is this the same to what was the as previous script doing?

I'll check it.